lib/spore/environment.rb

```diff
@@ -17,11 +17,20 @@ def read
     end
 
     def apply
-      each { |k, v| ENV[k] ||= v }
+      each { |k, v| ENV[k] ||= maybe_indirect(v) }
     end
 
     def apply!
-      each { |k, v| ENV[k] = v }
+      each { |k, v| ENV[k] = maybe_indirect(v) }
+    end
+
+    private
+
+    # Indirect values point to other environment variables and start with two
+    # tildes (~~).
```

Comment:
Is the double tilde `~~` a standard way to refer to other environment variables?